dotnet/src/Connectors/Connectors.UnitTests/Memory/Sqlite/SqliteMemoryStoreTests.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
+#if NET // TODO: Tests fail with "Library e_sqlite3 not found" on .NET Framework
```

Comment:
Seems related to this issue and the linked issues:
https://github.com/ericsink/SQLitePCL.raw/issues/252

Reply:
Seems like it's related to shadow copying and how SQLitePCLRaw.core.DLL is trying to load e_sqlite3.dll.
A workaround is referred to here, i.e. turn off shadow copying: https://github.com/dotnet/efcore/issues/19396.

When shadow copying is enabled for xunit, SQLitePCLRaw.core.DLL tries to load e_sqlite3.dll from a path relative to where the shadow copied SQLitePCLRaw.core.DLL resides, but the shadow copy folder structure doesn't match the build output folder structure, so it fails to load the dll.